sopel/modules/wikipedia.py

```diff
@@ -10,6 +10,7 @@
 from html.parser import HTMLParser
 import logging
 import re
+import urllib.parse
```

Comment:
D doesn't care, and I do too much. 😁 

```suggestion
from urllib.parse import urlparse
```

Reply:
Or, even:

```suggestion
from urllib.parse import urlparse, unquote
```

And then remove the import `from sopel.tools.web import unquote` which is really a useless tool.